src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.jsx

```diff
@@ -16,12 +18,29 @@ import WebexMeetingInfo from '../WebexMeetingInfo/WebexMeetingInfo';
  * @returns {object} JSX of the component
  */
 export default function WebexInterstitialMeeting({meetingID}) {
+  const {videoPermission} = useMeeting(meetingID);
+  const showModal = videoPermission === 'asking';
```

Comment:
when https://github.com/webex/component-adapter-interfaces/pull/62 is merged, you can use the value from the enum.